src/components/AllTextInputs.tsx

```diff
@@ -2,13 +2,14 @@ import TextInput from "./TextInput";
 import { GroupInfo, GroupInfoFieldNames } from "../utils/globalTypes";
 import { Box } from "@mui/material";
 import { fieldNames } from "../utils/globalVars";
+import { FC } from "react";
 
 interface Props {
   formData: GroupInfo;
   updateFormData: (e: React.ChangeEvent<HTMLInputElement>) => void;
 }
 
-const AllTextInputs = ({ formData, updateFormData }: Props) => {
+const AllTextInputs: FC<Props> = ({ formData, updateFormData }) => {
```

Comment:
I'd recommend this syntax for declaring react components.  For one, the previous signature didn't indicate at all that the method was a react component.  It just happened to take in those props, and it just happened to spit out JSX (also untyped previously), but using `FC` or `FunctionalComponent` should help give you some extra typed guardrails.

The other benefit is typing the deconstructed params can look kinda awkward

```js
({ formData, updateFormData }: Props)
```

And potentially get confusing if you need to rename during destructuring or provide default values

```js
({ formData, updateFormData : myFormUpdate}: Props)
```